src/import/FSHImporter.ts

```diff
@@ -1559,7 +1559,7 @@ export class FSHImporter extends FSHVisitor {
           // then, make all the replacements: closing parenthesis and comma
           return substrComma.replace(/\\\)/g, ')').replace(/\\,/g, ',');
         })
-        .filter(s => s); // Filter out any null values from incorrectly split escaped commas
+        .filter(s => s != null); // Filter out any null values from incorrectly split escaped commas
```

Comment:
This comment is confusing me a bit. This filtering is handling empty parameter values right?

Reply:
Oh ok, and we were incorrectly filtering the `''`, which messed with empty params, gotcha that does make sense.